test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -806,6 +819,364 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				Expect(err).ToNot(HaveOccurred())
 				Expect(cpuSection.Key("enabled").String()).To(Equal("false"))
 			})
+
+			It("[test_id:54178]Verify System is tuned when updating from HighPowerConsumption to PerPodPowermanagment", func() {
+
+				//First enable HighPowerConsumption
+				By("Modifying profile")
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					HighPowerConsumption:  pointer.Bool(true),
```

Comment:
This test override this one right? 
https://github.com/marioferh/cluster-node-tuning-operator/blob/5d9c2e11ad5595b7724edb327ed1a34e83518fc1/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go#L638

Reply:
Yes, this test we are checking if we can move from one Hint to another is the system tuned appropriately

Reply:
Then we can remove the previous one?